test/micro-distribution/lib/Standard/Base/0.0.0-dev/src/Panic.enso

```diff
@@ -1,4 +1,7 @@
+import project.Any.Any
+
 @Builtin_Type
 type Panic
     throw payload = @Builtin_Method "Panic.throw"
-    catch_primitive ~action handler = @Builtin_Method "Panic.catch_primitive"
+    catch panic_type ~action handler = @Builtin_Method "Panic.catch"
+    catch_primitive ~action handler = Panic.catch Any action handler
```

Comment:
Why do we still define `catch_primitive` here? Wasn't it removed from the normal distribution? Suspicious.

Reply:
Yes, it is removed from the distribution.
We had a few interpreter tests that used to do
```
Panic.catch_primitive @ caught-> caught.payload
```
which return a syntax error. Except, when translating to `Panic.catch Any @ caught-> caught.payload` they would fail to catch it. 

In reality the application is broken because of a broken IR.
It's a bug in the parser apparently (https://www.pivotaltracker.com/story/show/184054024) and I've been scratching my head around it for a while. In the interest of moving on I added this workaround, which can be obviously removed once the fix is in place.